packages/client/wallets/aa/src/blockchain/BlockchainNetworks.ts

```diff
@@ -288,25 +286,11 @@ export function getViemNetwork(networkName: EVMBlockchainIncludingTestnet): Chai
 }
 
 export function getBundlerRPC(chain: EVMBlockchainIncludingTestnet): string {
-    switch (chain) {
-        case EVMBlockchainIncludingTestnet.BASE_SEPOLIA:
-            return PM_BASE_SEPOLIA_RPC;
-        case EVMBlockchainIncludingTestnet.BASE:
-            return PM_BASE_RPC;
-        default:
-            return BUNDLER_RPC + getZeroDevProjectIdByBlockchain(chain) + "?bundlerProvider=STACKUP";
-    }
+    return BUNDLER_RPC + getZeroDevProjectIdByBlockchain(chain) + "?bundlerProvider=STACKUP";
 }
 
 export function getPaymasterRPC(chain: EVMBlockchainIncludingTestnet): string {
-    switch (chain) {
-        case EVMBlockchainIncludingTestnet.BASE_SEPOLIA:
-            return PM_BASE_SEPOLIA_RPC;
-        case EVMBlockchainIncludingTestnet.BASE:
-            return PM_BASE_RPC;
```

Comment:
hmm why delete this?
We have a paymaster from Base themselves to cover costs instead of us

Reply:
Sorry I just saw this. 

These RPCs didn't work, that's why I replaced them with the ones from ZeroDev. I tried to get more info about them but couldn't. Do you have some doc where I can get more info about them?